scripts/falco-driver-loader

```diff
@@ -278,31 +281,31 @@ clean_kernel_module() {
 	fi
 
 	KMOD_NAME=$(echo "${DRIVER_NAME}" | tr "-" "_")
-	if lsmod | cut -d' ' -f1 | grep -qx "${KMOD_NAME}"; then 
+	if lsmod | cut -d' ' -f1 | grep -qx "${KMOD_NAME}"; then
 		if rmmod "${DRIVER_NAME}" 2>/dev/null; then
-			echo "* Unloading ${DRIVER_NAME} module succeeded"
-		else 
-			echo "* Unloading ${DRIVER_NAME} module failed"
+			echo "* Unloading the Falco module succeeded"
+		else
+			echo "* Unloading the Falco module failed"
 		fi
 	else
-		echo "* No ${DRIVER_NAME} module loaded"
+		echo "* There is no Falco module loaded"
 	fi
 
-	if ! hash dkms &>/dev/null; then
+	if ! hash dkms >/dev/null 2>&1; then
 		echo "* Skipping dkms remove (dkms not found)"
 		return
 	fi
-    
+
 	DRIVER_VERSIONS=$(dkms status -m "${DRIVER_NAME}" | cut -d',' -f2 | sed -e 's/^[[:space:]]*//')
 	if [ -z "${DRIVER_VERSIONS}" ]; then
-		echo "* No ${DRIVER_NAME} module found in dkms"
+		echo "* There is no Falco module in dkms"
 		return
 	fi
 	for CURRENT_VER in ${DRIVER_VERSIONS}; do
 		if dkms remove -m "${DRIVER_NAME}" -v "${CURRENT_VER}" --all 2>/dev/null; then
-			echo "* Removing ${DRIVER_NAME}/${CURRENT_VER} succeeded"
-		else 
-			echo "* Removing ${DRIVER_NAME}/${CURRENT_VER} failed"
+			echo "* Removing the Falco module (version ${CURRENT_VER}) succeeded"
```

Comment:
I liked the old way this was printed, it was way more clear printing the actual name of the removed dkms module